scripts/symbolize.py

```diff
@@ -205,7 +205,7 @@ def resolve(self, addr):
         if not reladdr or not self._addr2line:
             return '???'
         try:
-            print >> self._addr2line.stdin, reladdr
+            print(reladdr, file=self._addr2line.stdin)
```

Comment:
I this really correct? `print >> self._addr2line.stdin, reladdr` saves `reladdr` in `addr2line.stdin`  I think? But `print(reladdr, file=self._addr2line.stdin)` just prints it.

Wouldn't 
```python
file=self._addr2line.stdin = reladdr
```
be the correct way to write this? Maybe the way it's done with the "`...stdout...`" right after, might make it work anyways. @jforissier you're name shows up in git blame, how are things supposed to work here?

Reply:
The old syntax was really confusing ,`>>` redirect output to a file, see [print documentation](https://docs.python.org/2/reference/simple_stmts.html#the-print-statement).
Because of the confusion this was removed in python3, where print is no longer a statement and the function takes the `file` argument to redirect the print output.

Reply:
I was reading that before posting my answer, still a bit pussled. So you're saying that this line
```python
print(reladdr, file=self._addr2line.stdin)
```
actually stores `reladdr` in `self._addr2line.stdin`?

Reply:
@jbech-linaro correct, in other words it sends some data to the child process.